mautic/mautic
# SugarCRM support for store fields to tags

**Please be sure you are submitting this against the _staging_ branch.**

[//]: # ( Please answer the following questions: )

| Q  | A
| --- | ---
| Bug fix? | 
| New feature? | 
| Automated tests included? |
| Related user documentation PR URL | 
| Related developer documentation PR URL | 
| Issues addressed (#s or URLs) | 
| BC breaks? | 
| Deprecations? | 

[//]: # ( Note that all new features should have a related user and/or developer documentation PR in their respective repositories. )

[//]: # ( Required: )
#### Description:
This PR added new option to store any fields from SugarCRM to contact tags.

![image](https://user-images.githubusercontent.com/462477/56050997-a2ff0e00-5d4d-11e9-810f-193700d315a9.png)


#### Steps to test this PR:
1. Load up [this PR](https://mautibox.com)
2. Setup Sugar CRM integration https://www.mautic.org/docs/en/plugins/sugarcrm.html
3. Setup lead sync and see in features tab new option Sugar CRM fields to Mautic tags 
4. Select some fields in new option SugarCRM support for store fields to tags (Unique identifier at least)
5. Try sync data by `php app/console mautic:integration:fetchleads --integration=Sugarcrm --env=dev  --time-interval="10 days"`
6. Check If contacts was added to Mautic and contacts has tags from setup above
